public/app/plugins/panel/alertlist/AlertList.tsx

```diff
@@ -0,0 +1,296 @@
+      <div className={styles.container}>
+        {noAlertsMessage && <div className={styles.noAlertsMessage}>{noAlertsMessage}</div>}
+
+        {props.options.showOptions === ShowOption.Current
+          ? !currentAlertState.loading &&
+            currentAlertState.value && (
+              <section>
+                <ol className={styles.alertRuleList}>
+                  {currentAlertState.value!.map((alert) => (
+                    <li className={styles.alertRuleItem} key={`alert-${alert.id}`}>
+                      <div className={cx(styles.alertRuleItemIcon, alert.stateModel.stateClass)}>
+                        <Icon name={alert.stateModel.iconClass} size="xl" className={styles.alertIcon} />
+                      </div>
+                      <div className={styles.alertRuleItemBody}>
+                        <div className={styles.alertRuleItemHeader}>
+                          <p className={styles.alertRuleItemName}>
+                            <a href={`${alert.url}?viewPanel=${alert.panelId}`}>{alert.name}</a>
+                          </p>
+                          <div className={styles.alertRuleItemText}>
+                            <span className={alert.stateModel.stateClass}>{alert.stateModel.text}</span>
+                            <span className={styles.alertRuleItemTime}> for {alert.newStateDateAgo}</span>
+                          </div>
+                        </div>
+                      </div>
+                    </li>
+                  ))}
+                </ol>
+              </section>
+            )
+          : !recentStateChanges.loading &&
+            recentStateChanges.value && (
+              <section>
+                <ol className={styles.alertRuleList}>
+                  {recentStateChanges.value.map((alert) => (
+                    <li className={styles.alertRuleItem} key={`alert-${alert.id}`}>
+                      <div className={cx(styles.alertRuleItemIcon, alert.stateModel.stateClass)}>
+                        <Icon name={alert.stateModel.iconClass} size="xl" />
+                      </div>
+                      <div className={styles.alertRuleItemBody}>
+                        <div className={styles.alertRuleItemHeader}>
+                          <p className={styles.alertRuleItemName}>{alert.alertName}</p>
+                          <div className={styles.alertRuleItemText}>
+                            <span className={alert.stateModel.stateClass}>{alert.stateModel.text}</span>
+                          </div>
+                        </div>
+                        <span className={styles.alertRuleItemInfo}>{alert.info}</span>
+                      </div>
+                      <div className={styles.alertRuleItemTime}>
+                        <span>{alert.time}</span>
+                      </div>
+                    </li>
+                  ))}
+                </ol>
+              </section>
+            )}
```

Comment:
```suggestion
        <ol className={styles.alertRuleList}>
          {props.options.showOptions === ShowOption.Current
            ? !currentAlertState.loading &&
              currentAlertState.value &&
              currentAlertState.value!.map((alert) => (
                <li className={styles.alertRuleItem} key={`alert-${alert.id}`}>
                  <Card heading={alert.name} href={`${alert.url}?viewPanel=${alert.panelId}`}>
                    <Card.Figure className={cx(styles.alertRuleItemIcon, alert.stateModel.stateClass)}>
                      <Icon name={alert.stateModel.iconClass} size="xl" className={styles.alertIcon} />
                    </Card.Figure>
                    <Card.Meta>
                      <div className={styles.alertRuleItemText}>
                        <span className={alert.stateModel.stateClass}>{alert.stateModel.text}</span>
                        <span className={styles.alertRuleItemTime}> for {alert.newStateDateAgo}</span>
                      </div>
                    </Card.Meta>
                  </Card>
                </li>
              ))
            : !recentStateChanges.loading &&
              recentStateChanges.value &&
              recentStateChanges.value.map((alert) => (
                <li key={`alert-${alert.id}`} className={styles.alertRuleItem}>
                  <Card heading={alert.alertName}>
                    <Card.Figure className={cx(styles.alertRuleItemIcon, alert.stateModel.stateClass)}>
                      <Icon name={alert.stateModel.iconClass} size="xl" />
                    </Card.Figure>
                    <Card.Meta>
                      <span className={alert.stateModel.stateClass}>{alert.stateModel.text}</span>
                      <span>{alert.time}</span>
                      {alert.info && <span className={styles.alertRuleItemInfo}>{alert.info}</span>}
                    </Card.Meta>
                  </Card>
                </li>
              ))}
        </ol>
```

Reply:
I wonder if we can try using the Card component here. Would allow to smplify the markup a bit and possible reduce the amount of styles we need to declare. Would need to reduce the padding of the card and add an option to customise `Figure` className.
![Screenshot 2021-03-11 at 10 08 11](https://user-images.githubusercontent.com/8878045/110755886-22efc480-8252-11eb-83ca-aee8ec619f67.png)
![Screenshot 2021-03-11 at 10 08 20](https://user-images.githubusercontent.com/8878045/110755891-2420f180-8252-11eb-8e6a-8999a92711f0.png)